apps/glusterfs/volume_durability_test.go

```diff
@@ -128,27 +128,27 @@ func TestDisperseDurability(t *testing.T) {
 	r.Data = 8
 	r.Redundancy = 3
 
-	gen := r.BrickSizeGenerator(200 * GB)
+	gen := r.BrickSizeGenerator(100 * GB)
```

Comment:
Same comment as with other test: leave the 200 here and add another block at the end. (Gen 5)